nova_vm/src/heap.rs

```diff
@@ -182,6 +182,11 @@ impl Heap {
             weak_maps: Vec::with_capacity(0),
             weak_refs: Vec::with_capacity(0),
             weak_sets: Vec::with_capacity(0),
+            // Drop scripts, modules, and eval sources last to ensure that all
+            // objects referring to them have dropped first.
+            eval_sources: Vec::with_capacity(0),
+            modules: Vec::with_capacity(0),
+            scripts: Vec::with_capacity(1),
```

Comment:
**issue:** Changing the order of fields when constructing `Heap` wouldn't do anything, right? I think you want to change them in the struct definition.

Reply:
Oops, yeah.